{
    "config": {
        "abort": {
            "already_configured": "Device is already configured"
        },
        "error": {
            "cannot_connect": "Failed to connect",
            "invalid_auth": "Invalid authentication",
            "unknown": "An unexpected error occurred. Please check the HomeAssistant logfiles."
        },
        "step": {
            "user": {
                "data": {
                    "email": "E-mail",
                    "password": "Password"
                }
            }
        }
    },
    "options": {
        "step": {
            "init": {
                "data": {
                    "tracing": "API response tracing. Requires debug logging enabled in configuration.yaml."
                }
            }
        }
    }
}
